.github/workflows/tests.yml

```diff
@@ -14,4 +14,4 @@ jobs:
         run: |
           python -m pip install --upgrade pip
       - name: Run tests
-        run: python tests/test_create.py
\ No newline at end of file
+        run: python tests/test_set.py
```

Comment:
You should run all test files. I would recommend to create a file call `test.py` that call all tests